Chummer/sheets/de-de/xz.language.xslt

```diff
@@ -309,6 +339,7 @@
   <xsl:variable name="lang.ResistDrain"    select="'Entzugswiderstand mit'"/>
   <xsl:variable name="lang.ResistFading"    select="'Schwundwiderstand mit'"/>
   <xsl:variable name="lang.RiggerInitiative"  select="'Rigger AR Initiative'"/>
+  <xsl:variable name="lang.SelectedGear"    select="'Ausgewähltes Zahnrad'"/>
```

Comment:
correct: "'Gang'"

Reply:
I think that 'Ausgewählte Ausrüsting' is the better/more corret translations.

Reply:
ahh ok, an misunderstanding.
in this context you are right of course.
"Gewählte Ausrüstung" would be evcen better, because it is shorter.
```suggestion
  <xsl:variable name="lang.SelectedGear"    select="'Gewählte Ausrüstung'"/>
```